packages/inventory/src/services/reservation-item.ts

```diff
@@ -131,38 +131,44 @@ export default class ReservationItemService {
    */
   @InjectEntityManager()
   async create(
-    data: CreateReservationItemInput,
+    data: CreateReservationItemInput | CreateReservationItemInput[],
     @MedusaContext() context: SharedContext = {}
-  ): Promise<ReservationItem> {
+  ): Promise<ReservationItem[]> {
```

Comment:
todo: should return single or collection? to keep consistency with the rest of the methods